RELEASING.md

```diff
@@ -1,5 +1,6 @@
 ## Releasing
 
+1. Rename the 'alpakka-xx-stable' report in [WhiteSource](https://saas.whitesourcesoftware.com) to reflect the latest version
```

Comment:
This seems to require extra permission. Should it be phrased "Request Akka team to ..."?

Reply:
👍 good one will do!